modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/chemistry_for_ERP.dm

```diff
@@ -343,245 +280,170 @@ Haha! Kill me please.
-
-				if(M.reagents.has_reagent(/datum/reagent/penis_enlarger))
-					M.reagents.remove_reagent(/datum/reagent/breast_enlarger, 20)
-
-			else
-				return
-
-		if(ISINRANGE_EX(P?.genital_size, 18, 20) && (H.w_uniform || H.wear_suit))
-			var/target = H.get_bodypart(BODY_ZONE_PRECISE_GROIN)
-			if(prob(20))
-				to_chat(H, span_danger("You feel the tight in your pants!</b>"))
-				H.apply_damage(1, BRUTE, target)
-
-	//If they've opted out, then route processing though liver.
-	if(!(H.client?.prefs.read_preference(/datum/preference/toggle/erp/penis_enlargement)))
-		var/obj/item/organ/liver/L = H.getorganslot(ORGAN_SLOT_LIVER)
-		if(L)
-			L.applyOrganDamage(0.25)
-		else
-			H.adjustToxLoss(1)
-		return..()
-	return ..()
+	var/obj/item/organ/genital/penis/mob_penis = exposed_mob.getorganslot(ORGAN_SLOT_PENIS)
+	exposed_mob.penis_enlarger_amount += 5
+	if(exposed_mob.penis_enlarger_amount >= 100)
+		if(mob_penis?.genital_size > 20)
+			return ..()
+		mob_penis.genital_size += 1
+		if(prob(20) && mob_penis.girth < 20)
+			mob_penis.girth +=1
+		mob_penis.update_sprite_suffix()
+		exposed_mob.update_body()
+		exposed_mob.penis_enlarger_amount = 0
+
+	if(ISINRANGE_EX(mob_penis?.genital_size, 18, 20) && (exposed_mob.w_uniform || exposed_mob.wear_suit))
+		var/target = exposed_mob.get_bodypart(BODY_ZONE_PRECISE_GROIN)
+		if(prob(20))
+			to_chat(exposed_mob, span_danger("You feel a tightness in your pants!"))
+			exposed_mob.apply_damage(1, BRUTE, target)
 
-/datum/reagent/penis_enlarger/overdose_process(mob/living/carbon/human/M) //Turns you into a male if female and ODing, doesn't touch nonbinary and object genders.
-	if(!istype(M))
-		return ..()
-	// let's not kill them if they didn't consent.
-	if(!(M.client?.prefs.read_preference(/datum/preference/toggle/erp/gender_change)))
-		return..()
-
-	var/obj/item/organ/genital/breasts/B = M.getorganslot(ORGAN_SLOT_BREASTS)
-	var/obj/item/organ/genital/vagina/V = M.getorganslot(ORGAN_SLOT_VAGINA)
-	var/obj/item/organ/genital/womb/W = M.getorganslot(ORGAN_SLOT_WOMB)
-
-	if(M.gender == FEMALE)
-		M.set_gender(MALE)
-		M.body_type = M.gender
-		M.update_body()
-		M.update_mutations_overlay()
-
-	if(B)
-		if(B.genital_size >= 3)
-			B.genital_size -=2
-		else
-			return
-	if(V)
-		if(V.genital_size >= 3)
-			V.genital_size -=2
-		else
-			return
-	if(W)
-		if(W.genital_size >= 3)
-			V.genital_size -=2
-		else
-			return
 	return ..()
 
+/datum/reagent/drug/aphrodisiac/penis_enlarger/overdose_effects(mob/living/carbon/human/exposed_mob) //Turns you into a male if female and ODing, doesn't touch nonbinary and object genders.
+	. = ..()
+
+	var/obj/item/organ/genital/breasts/mob_breasts = exposed_mob.getorganslot(ORGAN_SLOT_BREASTS)
+	if(exposed_mob.gender == FEMALE)
+		exposed_mob.set_gender(MALE)
+		exposed_mob.body_type = exposed_mob.gender
+		exposed_mob.update_body()
+		exposed_mob.update_mutations_overlay()
+
+	if(mob_breasts.genital_size > 2)
+		mob_breasts.genital_size -=2
```

Comment:
Add a space between the number and the symbol. Also, could use being a define, methinks.